src/python/CMakeLists.txt

```diff
@@ -110,7 +110,7 @@ if (PYTHONLIBS_FOUND)
 
     foreach (test ${python_tests})
       add_test(NAME ${test}.py COMMAND
-        python3 ${CMAKE_SOURCE_DIR}/src/python/${test}.py)
+        "${Python3_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/src/python/${test}.py")
```

Comment:
which cmake call is providing this variable?

Reply:
`find_package(Python3)` here https://github.com/ignitionrobotics/ign-math/pull/239/files#diff-43c9042647d14a73e4061f77bba19ee368fd059a7e0c6086063f7646e2b2aabbR69. Umm, that call is conditionally scoped, I thought it was used in the root CMakeLists.txt but it is the old `find_package(PythonLibs)`. We don't have `find_package(Python3)` in Bionic, I changed the code to use old `PYTHON_EXECUTABLE` variable.